src/cliClient.js

```diff
@@ -53,7 +53,7 @@ cli.CommandLineClient = class extends BaseClient {
 	 * The net library listens on the event called `data`
 	 */
   addMessageListener(listener) {
-    this.on('message',(data)=>{ listener(data,this)});
+    this.on('data',(data)=>{ listener(data,this)});
```

Comment:
weird, did this ever work for us before with `message`? or is this another difference between TCP socket and websocket?

Reply:
I actually had made a change to this in the previous commit that changed it from 'data' to 'message'. Fixed it.

Reply:
lol, okay that makes sense. thought i was imagining...